File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L5/01/L5.py
import sys

def print_array(arr):
    for i in range(len(arr)-1, -1, -1):
        print(arr[i])

def main():
    angka = [0] * 10
    for i in range(10):
        print("Read a number: ", end="")
        angka[i] = int(input())
    
    print_array(angka)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-06/plagiarized/L5/01/L5.ts
import { readlineSync } from 'readline-sync';

function print(arr: number[]): void {
  let i = 9;
  while (i >= 0) {
    console.log(arr[i]);
    i--;
  }
}

function main(): void {
  const angka: number[] = new Array(10);
  let i = 0;
  while (i !== 10) {
    console.log('Read a number: ');
    angka[i] = Number(readlineSync.question());
    i++;
  }

  print(angka);
}

main();

